docs/cloud/pouta/launch-vm-from-web-gui.md

```diff
@@ -157,6 +157,20 @@ Click the plus to create a new IP, select the IP, select your machine under **In
 **Figure** Floating IP association dialog
 
 !!! Tip
-    Please note: Allocated and assigned floating IPs are billed at the rate of 0,2 BU/hr. You can additionally read our [blog post](http://pouta.blog.csc.fi) for management of floating IPs in a cPouta project.
+    Please note: Allocated and assigned floating IPs are billed at the rate of 0,2 BU/hr. You can additionally read our [blog post](http://cloud.blog.csc.fi/2017/12/floating-ip-management.html) for management of floating IPs in a cPouta project.
+
+## Adding a security groups to the machine in cPouta
+
+In the project, a user can manage the security group by creating or updating existing ones. A security group can be updated by adding different security rules for allowing or preventing network traffics. 
+
+![Create a security group](/img/create_security_group.png)
+
+**Figure** Creating and managing security groups
+
+The new security group can be added to different virtual machines. A virtual machine can have many security groups. Different security groups can be added or removed from idfferent VMs. 
```

Comment:
idfferent -> different

Also the images seem to be empty/0 byte

Reply:
Thanks

Reply:
"The new security group can be added to different virtual machines. A virtual machine can have many security groups. Different security groups can be added or removed from idfferent VMs."

How about if we write these three lines like bellow:
"A new security group can be added to other VMs in the same project. On the other hand, we can also remove a security group from running VM."

Reply:
Thanks.

Reply:
Done